test/validator/samples/a11y-aria-proptypes-tokenlist/input.svelte

```diff
@@ -3,5 +3,5 @@
 <div aria-relevant />
 <div aria-relevant={true} />
 <div aria-relevant={"false"} />
-<div aria-relevant="additions removalss" />
-<div aria-relevant="additions removalss " />
\ No newline at end of file
+<div aria-relevant="additions removals" />
+<div aria-relevant="additions removals " />
```

Comment:
I try not to change lines beyond the contents I'm changing, but Git, GitHub and I (personally) don't like files that don't have a newline at end of file.